example/spec/parallel/discoverGranules/DiscoverGranulesS3FailureSpec.js

```diff
@@ -0,0 +1,96 @@
+'use strict';
+
+const pWaitFor = require('p-wait-for');
+const { randomString } = require('@cumulus/common/test-utils');
+const {
+  buildAndExecuteWorkflow,
+  loadCollection,
+  loadProvider
+} = require('@cumulus/integration-tests');
+const {
+  createCollection, deleteCollection
+} = require('@cumulus/integration-tests/api/collections');
+const { getExecution } = require('@cumulus/integration-tests/api/executions');
+const {
+  createProvider, deleteProvider
+} = require('@cumulus/integration-tests/api/providers');
+const { loadConfig } = require('../../helpers/testUtils');
+
+describe('The DiscoverGranules workflow with a non-existent bucket', () => {
+  let beforeAllCompleted = false;
+  let collection;
+  let provider;
+  let workflowExecution;
+  let stackName;
+  let bucket;
+
+  beforeAll(async () => {
+    ({ stackName, bucket } = await loadConfig());
+
+    const testId = randomString();
+
+    // Create the provider
+    provider = await loadProvider({
+      filename: './data/providers/s3/s3_provider.json',
+      postfix: testId,
+      s3Host: randomString()
```

Comment:
So this is what causes the workflow failure? The use of a non-existent bucket?

Reply:
Correct. I asked Chuck about it (since he wrote the ticket), and he was happy with that for the failure test.